index.html

```diff
@@ -105,19 +109,23 @@ <h2>Terminology</h2>
       </p>
       <ul>
         <li>
-          <dfn data-lt="media timed event">media timed events</dfn> &mdash;
-          metadata events synchronized to the <a>media timeline</a> of a
-          <a>media resource</a>.
+          <dfn data-lt="timed metadata">timed metadata</dfn> &mdash;
+          metadata that is associated with a point in time or a region of time,
```

Comment:
It's often tempting to use positional words in place of temporal ones, but since temporal equivalents usually exist I think it's clearer to use them. Editorial suggestions:

* s/point/moment
* s/region/period

Reply:
I have changed 'region' to 'period', but point is fine IMO.